completions/bash/multipass

```diff
@@ -87,6 +90,7 @@ _multipass_complete()
                 _filedir
                 return
             ;;
+            # TODO: add "--network" here
```

Comment:
Of course, we should do it. Instead of opening an issue, I added a card to the bridging project: https://github.com/canonical/multipass/projects/12#card-46917084

Reply:
Ok, if you prefer it that way. In any case, notice that issue doesn't mean bug, it's really just a ticket that can also track planned features.

Reply:
I agree it's not a bug and it's a very useful feature. But I didn't want to add an issue for a yet unreleased code. I'd prefer to promote the card to issue and link them once the code lands in master.

Reply:
OK, if you prefer it that way. I think it is fine to have issues tracking tasks that need to happen only after other tasks are completed. But this is not that big of a deal, so I am OK with the card instead.